packages/components/src/components/themed-scrollbars/themed-scrollbars.jsx

```diff
@@ -7,7 +7,6 @@ const ThemedScrollbars = ({
     children,
     className,
     has_horizontal,
-    height,
```

Comment:
This component has been used in many areas of the application, this will have side-effects. Why do we need to remove the prop here?

Reply:
I reverted these changes now and put the styling in the reports.scss file to only specify changes to the statements page. I have to use !important statements here to be able to overwrite some of the JS styling.